olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/StreamLayerClient.h

```diff
@@ -199,6 +199,34 @@ class DATASERVICE_READ_API StreamLayerClient final {
   client::CancellableFuture<DataResponse> GetData(
       const model::Message& message);
 
+  /**
+   * @brief Read messages from stream layer and commits the successfully
+   * consumed messaged before handing them over to the user
+   *
+   * Only possible if Subscribe() was called before.
```

Comment:
```suggestion
   * Only possible if subscribed successfully.
```
Would that read more straightforward?